packages/ckeditor5-engine/src/view/domconverter.ts

```diff
@@ -182,7 +182,7 @@ export default class DomConverter {
 			'td', 'tfoot', 'th', 'thead', 'tr', 'ul'
 		];
 		this.inlineObjectElements = [
-			'object', 'iframe', 'input', 'button', 'textarea', 'select', 'option', 'video', 'embed', 'audio', 'img', 'canvas'
```

Comment:
Why this change was needed (what does it do?), and, also, it looks a bit weird, as we define DOM elements here, and `mergeField` is not a DOM element.

Also, as @niegowski commented, shouldn't it be injected from the feature? (I don't know this part of the code very well.)